app/views/casework/crime_applications/sections/_properties.html.erb

```diff
@@ -0,0 +1,155 @@
+<% if properties %>
+  <h2 class="govuk-heading-m">
+    <%= label_text(:properties_title) %>
+  </h2>
+
+  <% if properties.any? %>
+    <% properties.each_with_index do |property| %>
+      <% asset = PropertiesPresenter::PROPERTY_TYPE_MAPPING[property.property_type.to_sym][:display_name]%>
+      <dl class="govuk-summary-list govuk-!-margin-bottom-9">
```

Comment:
@ripan might be worth checking out my PR. I've added a component for rendering grouped lists of things:

https://github.com/ministryofjustice/laa-review-criminal-legal-aid/pull/567/files#diff-7ea3a2f1a2bb8203307514304b03264f22a77054911e013cd8f645016dd2f379R1-R22

Reply:
UI fixed with new components